composer.json

```diff
@@ -19,6 +18,7 @@
         "drupal/config_devel": "~1.2",
         "drupal/console": "^1",
         "drupal/drupal-extension": "^4.0",
+        "drupal/styleguide": "~1.0-alpha3",
```

Comment:
We should keep this alphabetically ordered.

Reply:
The alphabetically ordered is respected.

Reply:
facepalm.